src/app/public/modules/flyout/flyout.component.spec.ts

```diff
@@ -196,6 +201,7 @@ describe('Flyout component', () => {
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [
+        RouterModule,
```

Comment:
Instead of importing the `RouterModule` into the `TestBed`, do you mind importing the `RouterTestingModule` into the `SkyFlyoutFixturesModule`?

Reply:
it's actually already there! removing my addition